words/objects.txt

```diff
@@ -862,27 +1100,41 @@ front
 frost
 frown
 fruit
+fruitadens
+fuchsia
 fuel
+fukuiraptor
 fur
 furniture
+fuschia
 galaxy
+galette
 galleon
 galley
 galliform
+gallimimus
+gallium
 gallon
 gambler
 game
 gander
 garage
 garden
+gardenia
 gargoyle
+gargoyleosaurus
 garlic
+garment
 garnet
 gas
 gasoline
+gasosaurus
+gastonia
 gate
 gateway
+gatsby
```

Comment:
is this something besides the a last name?

Reply:
another name for a newsboy cap! https://www.gentlemansgazette.com/flat-cap-newsboy-hat/

Reply:
T.I.L.